src/main/java/com/minecolonies/coremod/network/messages/OpenInventoryMessage.java

```diff
@@ -160,6 +160,7 @@ private static void doCitizenInventory(final OpenInventoryMessage message, final
             {
                 citizen.getInventoryCitizen().setCustomName(message.name);
             }
+            //player.displayGUIChest(new IItemHandlerToIInventoryWrapper(citizen.getInventoryCitizen(), citizen.getInventoryCitizen()));
             player.displayGUIChest(citizen.getInventoryCitizen());
```

Comment:
No it is their so it reminds me that i need to update the GUI when I update the InventoryCitizen.

Should probably make this a TODO.